docs/dev/testing.md

```diff
@@ -302,6 +302,57 @@ suite is as solid as possible. The first step is contributing a stable (read: no
 To do so, when developing tests, please run them (1) in debug mode and (2) 100 times in a row (using `--repeat 100`),
 and see that they pass successfully.
 
+## Allure reporting
+
+To make analyzing of the test results more convenient, an allure reporting tool is introduced.
+Python module allure-pytest module is included in the toolchain image and additional parameter added to gather allure
```

Comment:
Nit: `module allure-pytest module` - "module" unnecessarily appears twice.

Reply:
"...and additional parameter added..." -> "...and __an__ additional parameter __is__ added..."